eng/Publishing.props

```diff
@@ -1,7 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project>
-   <PropertyGroup>
-      <PublishingVersion>3</PublishingVersion>
-      <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
-   </PropertyGroup>
-</Project>
+  <PropertyGroup>
+    <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ItemsToPushToBlobFeed Include="$(ArtifactsPackagesDir)Release\*.nupkg"
+                           IsShipping="true"
+                           UploadPathSegment="Roslyn-analyzers"
```

Comment:
I don't see Arcade respecting the `UploadPathSegment` item metadata. Why did you have to set it?